locales/es/translation.json

```
@@ -0,0 +1,250 @@
+        "language": "Languaje"
+    },
+    "$Menu": {
+        "about": "Sobre",
+        "all-clear": "¡Todo despejado!",
+        "all-files": "Todos los archivos",
+        "cancel": "Cancelar",
+        "check-for-updates": "Búsqueda de actualizaciones",
+        "clear-database": "Borrar la base de datos",
+        "confirm-clear-all-data": "¿Estás seguro de que quieres borrar todos los datos?",
+        "confirm-import-db": "¿Estás seguro de que quieres importar una base de datos? Anularás cualquier información actual.",
+        "confirm-migrate-to-flexible": "¿Estás seguro de que quieres migrar la base de datos del calendario fijo al calendario flexible?\n\nLa base de datos del calendario flexible existente será eliminada.",
+        "copy": "Copiado",
+        "could-not-be-loaded": "no pudo ser cargado",
+        "cut": "Cortar",
+        "database-export": "Exportación de la base de datos",
+        "database-imported": "Base de datos importada",
+        "database-migrated": "Base de datos migrada",
+        "database-was-exported": "Bien, la base de datos fue exportada.",
+        "edit": "Editado",
+        "exit": "Salir",
+        "export-database": "Exportar la base de datos",
+        "export-db-to-file": "Exportar la base de datos a un archivo",
+        "export": "Exportar",
+        "failed-entries": "Entradas fallidas",
+        "failed-migrating": "Migración fallida",
+        "help": "Ayuda",
+        "import-database": "Importar base de datos",
+        "import-db-from-file": "Importar base de datos de un archivo",
+        "import-successful": "¡Sí! ¡Importación exitosa!",
+        "import": "Importar",
+        "menu": "Menú",
+        "migrate-calendar-to-flexible": "Migrate fixed calendar database to flexible",
+        "migrate-to-flexible-calendar": "Migrate to flexible calendar",
+        "migration-successful": "Sí! Migration successful!",
+        "no-thanks": "No, gracias",
+        "ok": "OK",
+        "paste": "Pegar",
+        "preferences": "Preferencias",
+        "punch-time": "Hora del golpe",
+        "punched-time": "Tiempo fichado",
+        "quit": "Salir",
+        "reload": "Recargar",
+        "select-all": "Seleccionar Todo",
+        "send-feedback": "Envíe sus comentarios",
+        "should-migrate-to-flexible": "Tu calendario flexible está vacío. ¿Quieres empezar por migrar la base de datos del calendario fijo existente a la flexible?",
+        "show-app": "Mostrar Aplicacion",
+        "something-went-wrong": "Algo malo pasó :(",
+        "toggle-dev-tools": "Alternar a herramientas de desarrollo",
+        "ttl-github": "TTL GitHub",
+        "view": "Vista",
+        "workday-waiver-manager": "Gerente de Exención de Jornada Laboral",
+        "yes-please": "Sí, por favor"
+    },
+    "$FlexibleDayCalendar": {
+        "switch-view": "Cambiar la vista",
+        "previous-day": "Día anterior",
+        "next-day": "Día siguiente",
+        "time-to-leave": "Hora de irse",
```

Time to leave is the name of the App, because of this we have not been translating it:

```suggestion
        "time-to-leave": "Time to Leave",
```